include/grpc/impl/codegen/grpc_types.h

```diff
@@ -142,6 +142,8 @@ typedef struct {
 /** Secondary user agent: goes at the end of the user-agent metadata
     sent on each request */
 #define GRPC_ARG_SECONDARY_USER_AGENT_STRING "grpc.secondary_user_agent"
+/** The maximum time between subsequent connection attempts, in ms */
```

Comment:
I think that there is a use case for subsecond retries, e.g. between processes on the same machine. 1 second might feel sluggish there. Anyway, your call.